docs/readthedocs/source/doc/Nano/Overview/known_issues.md

```diff
@@ -0,0 +1,53 @@
+# Nano Known Issues
+
+## **PyTorch Issues**
+
+### **AttributeError: module 'distutils' has no attribute 'version'**
+
+This usually is because the latest setuptools does not compatible with PyTorch 1.9.
+
+You can downgrade setuptools to 58.0.4 to solve this problem.
```

Comment:
can we specify the version in nano dependence?

Reply:
When a user creates a conda environment and specifics the python version, setuptools will be installed by conda. 

If we list `setuptools` as a dependency in pip, there might be conflict with conda.